frame/society/src/migrations.rs

```diff
@@ -40,24 +40,29 @@ impl<
 {
 	#[cfg(feature = "try-runtime")]
 	fn pre_upgrade() -> Result<Vec<u8>, TryRuntimeError> {
-		ensure!(can_migrate::<T, I>(), "pallet_society: already upgraded");
-
-		let current = Pallet::<T, I>::current_storage_version();
-		let onchain = Pallet::<T, I>::on_chain_storage_version();
-		ensure!(onchain == 0 && current == 2, "pallet_society: invalid version");
+		log::info!(target: TARGET, "pre_upgrade");
+		if !can_migrate::<T, I>() {
+			log::warn!(target: TARGET, "Already migrated");
```

Comment:
```suggestion
			log::warn!(target: TARGET, "pallet_society::MigrateToV2: Already migrated");
```

Makes it easier to find which one is already migrated

Reply:
isn't the purpose of `TARGET` to provide this kind of information?

Reply:
But `TARGET` is not shown by default. However, this is a valid argument and someone can just enable the target to be shown when running the try runtime stuff.

Reply:
ah I see. I'll add it anyway, since the context of 'already migrated' is always useful to know.